frontend/src/components/ComboboxInput.vue

```diff
@@ -131,6 +131,9 @@ const keydown = () => {
     focusedItemIndex.value < currentlyDisplayedItems.value.length
   ) {
     focusedItemIndex.value += 1
+  } else if (createNewItem.value) {
+    // In case we create a new item we need to focus on the child as it contains the button
+    focusedItemIndex.value += 2
```

Comment:
This i kind of brittle, to set the focus by a fixed number. @kiwikern and me came up with a little suggestion and it is a little bit more, we put it in a show and tell PR 

https://github.com/digitalservicebund/ris-backend-service/pull/1625